sites/public/pages/finder.tsx

```diff
@@ -0,0 +1,211 @@
+import {
+  encodeToFrontendFilterString,
+  FrontendListingFilterStateKeys,
+  ListingFilterState,
+} from "@bloom-housing/shared-helpers"
+import {
+  AppearanceStyleType,
+  Button,
+  Field,
+  Form,
+  FormCard,
+  ProgressNav,
+  StepHeader,
+  t,
+} from "@bloom-housing/ui-components"
+import axios from "axios"
+import router from "next/router"
+
+import React, { useEffect, useState } from "react"
+import { useForm } from "react-hook-form"
+import Layout from "../layouts/application"
+
+const getTranslationString = (str: string) => {
+  if (str === "studio") {
+    return "studioPlus"
+  } else if (str === "oneBdrm") {
+    return "onePlus"
+  } else if (str === "twoBdrm") {
+    return "twoPlus"
+  } else if (str === "threeBdrm") {
+    return "threePlus"
+  } else if (str === "fourBdrm") {
+    return "fourPlus"
+  }
+}
+
+interface finderField {
+  label: string
+  translation: string
+  selected: boolean
+}
+
+interface finderQuestion {
+  fieldGroupName: string
+  fields: finderField[]
+}
+
+interface finderForm {
+  [key: number]: finderQuestion
+}
+
+const Finder = () => {
+  const [currentStep, setCurrentStep] = useState<number>(1)
+  const [completedSteps, setCompletedSteps] = useState<number>(0)
```

Comment:
Is completedSteps always going to be just one less than currentStep? If so, I think you could get away with just having one `useState` for currentStep and then everywhere completedSteps is used just do `currentStep - 1`